### PR TITLE
Add a simple README.md file

### PR DESCRIPTION
**Pull request by XenofonV**
Adding a README file that was missing from the project

### CODE REVIEW

**Comment by GeorgeVasilopoulos** [comment]
This README is too short it would be nice to see an example.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Rearrange
+=========
+
+This module is used for rearranging names.
+Turns "LastName, FirstName" into "FirstName, LastName"
```
Comment (GeorgeVasilopoulos) [inline README.md]: The sentence is missing a period at the end

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Rearrange
+=========
+
+This module is used for rearranging names.
+Turns "LastName, FirstName" into "FirstName, LastName"
+
+# Example
```
Comment (GeorgeVasilopoulos) [inline README.md]: Please make this ## so that it's a smaller title

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Rearrange
+=========
+
+This module is used for rearranging names.
+Turns "LastName, FirstName" into "FirstName, LastName"
+
+# Example
+
+Calling `rearrange_name("Turing, Alan")` will return `"Alan Turing"`
```
Comment (GeorgeVasilopoulos) [inline README.md]: This is cool, but add a couple more examples.

**Review by GeorgeVasilopoulos — COMMENTED** [review]
Thanks! Please have a look at the comments

**Comment by XenofonV** [comment]
All comments resolved, please take another look

**Pull request merged**